PowerPointLabs/PowerPointLabs/ELearningLab/Views/ELearningWorkspaceViews/ELearningLabMainPanel.xaml.cs

```diff
@@ -73,11 +93,12 @@ public void ListViewItemPropertyChanged(object sender, PropertyChangedEventArgs
         public void HandleELearningPaneSlideSelectionChanged()
         {
             PowerPointSlide _slide = this.GetCurrentSlide();
-            if (_slide.ID.Equals(slide.ID))
+            if (IsSlideExist(slideId) && _slide.ID.Equals(slide.ID))
             {
                 return;
             }
             slide = _slide;
+            slideId = slide.ID;
```

Comment:
Better to add some comments here to say that this is to only update the items when slide still exist and current slide is changed (?).

Reply:
Could be a bit difficult to understand this function if the reader of the code did not follow the github discussion, thus comments might help.

Reply:
Thanks for the suggestion, comments added.